### PR TITLE
Health Monitor Readme + Generated CLI Docs

### PR DESCRIPTION
**Pull request by tasherif-msft**

### CODE REVIEW

File: doc/blobfuse2.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## blobfuse2
```
Comment (gapra-msft) [inline doc/blobfuse2.md:1]: Should we add this to the release pipeline so it automatically does this for us?

Comment (tasherif-msft) [reply doc/blobfuse2.md:1]: Seems unnecessary. These parameters are not going to be changing on each release

Comment (vibhansa-msft) [inline doc/blobfuse2.md:1]: Do we really need to checkin all these md files ?

**Review by vibhansa-msft — APPROVED** [review]

**Review by gapra-msft — APPROVED** [review]

File: doc/blobfuse2.md (hunk 1 above)
Comment (tasherif-msft) [reply doc/blobfuse2.md:1]: Yea doesn't hurt to keep them. Customers might also just go through these if they didn't access the MS Docs

**Pull request merged**